### ISSUE TITLE
idea:stroke to mesh

### ISSUE THREAD

**Issue by figihiuy** [chsh2/nijiGPen#5]
I want the resulting meshes to be automatically separated into new collections.

**Comment by chsh2** [comment]
Yes I am thinking of adding some methods organizing the generated objects. However, I have not decided the way of doing it. I may need to collect some user experience. Would you mind briefly talking about what kind of artworks you want to create with the stroke-to-mesh function?

For this specific issue, you can just click to select another collection in the outliner before hitting the add-on button. In this way, all generated objects will go to the new collection.
![image](https://user-images.githubusercontent.com/110356534/201510384-d4644240-8283-4437-be18-e5dd20964048.png)

**Comment by figihiuy** [comment]
It would certainly be nice to manually move it to the collection immediately after generating it. However, if not, if you don't like what you generated, delete it and regenerate it.
That iteration is a bit cumbersome.　

(Translate with a translator

**Comment by chsh2** [comment]
In the release v0.2.2, shortcuts were added to hide or delete all generated meshes from a GPencil object. I am still thinking of better ways of manage these meshes. Please let me know if you have any other ideas.
